docs/prepare-compressed-backup.md

```diff
@@ -1,20 +1,38 @@
 # Decompress and prepare a compressed backup
 
-Before you can prepare the backup you’ll need to uncompress all the files.
-*Percona XtraBackup* has implemented `--decompress` option
-that can be used to decompress the backup.
+Before you can prepare the backup you need to decompress all the files.
 
-```{.bash data-prompt="$"}
-$ xtrabackup --decompress --target-dir=/data/compressed/
-```
+## Decompress a backup
+
+* To decompress backups made using `LZ4` or `ZSTD` compression algorithm, install the corresponding package.
+
+    === "Install the `lz4` package"
+
+        ```{.bash data-prompt="$"}
+        $ sudo apt install lz4
+        ```
+
+    === "Install the `zstd` package"
 
-!!! note
+        ```{.bash data-prompt="$"}
+        $ sudo apt install zstd
```

Comment:
Please add the commands for RHEL/CentOS variants too. `yum install..`

Reply:
Added